source/Sylvan.Data.Excel.Tests/Sylvan.Data.Excel.Tests.csproj

```diff
@@ -3,35 +3,23 @@
 	<PropertyGroup>
 		<TargetFrameworks>net8.0;net48</TargetFrameworks>
 		<RootNamespace>Sylvan.Data.Excel</RootNamespace>
-		<IsPackable>false</IsPackable>
-		<DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
-		
-		<!-- 
-		Suppress nuget vulnerability warnings, as I'm not really concerned about these in the test project.
-		Maybe I should be?
-		-->
-		<NoWarn>$(NoWarn);NU1903;xUnit2004</NoWarn>
 	</PropertyGroup>
 
 	<PropertyGroup Condition="$(TargetFramework) == 'net8.0'">
 		<DefineConstants>$(DefineConstants);ASYNC;DATEONLY</DefineConstants>
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Sylvan.BuildTools.Resources" Version="0.6.2">
-			<PrivateAssets>all</PrivateAssets>
-			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-		</PackageReference>
+		<PackageReference Include="Sylvan.BuildTools.Resources" Version="0.6.2" PrivateAssets="all" />
 		<PackageReference Include="Sylvan.Data" Version="0.2.16" />
-		<PackageReference Include="Sylvan.Data.Csv" Version="1.3.9" />
-		<PackageReference Include="System.Data.SqlClient" Version="4.9.0" />
+		<PackageReference Include="Sylvan.Data.Csv" Version="1.4.2" />
 		<ProjectReference Include="..\Sylvan.Data.Excel\Sylvan.Data.Excel.csproj" />
 		<Reference Include="System.IO.Compression" Condition="$(TargetFramework) == 'net48'" />
 	</ItemGroup>
 
 	
 	<ItemGroup>
-		<StaticResourceFolder Include="TestData" />
+		<StaticResourceFolder Include="TestData" Visible="false" />
```

Comment:
What's this for?

Reply:
To get rid of the extra `TestData` directory, visible at least in Rider.

<img width="404" alt="Screenshot 2025-05-01 at 20 53 44" src="https://github.com/user-attachments/assets/a01276c5-01dc-4c81-847a-b5f85b118b93" />

I think this could probably get fixed in the `Sylvan.BuildTools.Resources` package itself (assuming that's the package that handles the `StaticResourceFolder` items) by using an [ItemDefinitionGroup](https://learn.microsoft.com/en-us/visualstudio/msbuild/itemdefinitiongroup-element-msbuild), like I did in Chisel: https://github.com/0xced/Chisel/blob/dda8dad31c5573290fd29c2521b716fe63f25f36/src/Chisel/build/Chisel.props#L19-L23

Reply:
Thanks. That extra folder doesn't show up in VS. I'll merge this, and make a note to fix it in the Sylvan.BuildTools.Resources package too.